scripts/build_module.js

```diff
@@ -0,0 +1,6 @@
+const { parentPort, workerData } = require('worker_threads');
+const execa = require('execa');
+
+execa
+  .shell(`cd ${workerData.location} && npm run build --if-present`)
```

Comment:
Recommended to prefer using `execa()`.

Reply:
command here, i don’t use a file (and I can’t ? ^^)